lib/elements/prompt.js

```diff
@@ -15,6 +15,8 @@ class Prompt extends EventEmitter {
   constructor(opts={}) {
     super();
 
+    this.margin = (opts.margin > 0) ? opts.margin - 1 : 0;
```

Comment:
This is one part of why the current solution is probably more of a workaround.
I'm encountering the issue that the actual amount of spaces applied is always one higher than specified by the margin option.